lib/bedrock.js

```diff
@@ -138,6 +138,7 @@ api.start = async options => {
   try {
     await _runWorker(startTime);
   } catch(err) {
+    console.error('An error occurred during worker initialization.', err);
```

Comment:
I think logging is initialized at this point?  Would it make sense to do something like `loggers.get('app').error(...)` or is just plain console logging better in this case?

Reply:
@davidlehn if you would like to make this nice, it would be great.  Can you take this over?

Reply:
as long as the solution is logged to the console I'm happy, if it goes in bedrock logs as well, that's a bonus.